.yam.yaml

```diff
@@ -0,0 +1,6 @@
+gap:
+  - "."
+  - ".subpackages"
+  - ".data"
```

Comment:
This is based on the most common formatting I saw among our config files. **Very open to feedback!** 😃 

🤔  Should we also add an item for `.pipeline`? Just eyeballing, it seemed roughly 50/50 on configs that had blank lines between pipeline items.

Reply:
I like the idea of using `.pipeline` :)

Reply:
Added!